data/json/items/tools.json

```diff
@@ -5838,7 +5843,7 @@
     "description": "This is a sealed glass jar containing sauerkraut.  Use to open and eat to enjoy.",
     "price": 650,
     "material": ["glass", "veggy"],
-    "weight": 550,
+    "weight": 1750,
```

Comment:
Well maybe it's not the weight here that needs to be adjusted then. Doesn't make sense to match the wonky weight by making other things also wonky.